xrtl/gfx/pixel_diff_test.cc

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "xrtl/testing/gtest.h"
+#include "xrtl/testing/render_test.h"
+
+namespace xrtl {
+namespace gfx {
+namespace {
+
+// Checks that pixel diffing render tests work.
+class PixelDiffTest : public testing::RenderTest {};
+
+TEST_F(PixelDiffTest, SameImage) {
```

Comment:
move these side-by-side with the CompareImages code (so if you had testing/pixel_diff.h/cc, this would be pixel_diff_test.cc)